src/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.4'
+
+services:
```

Comment:
It is good practice to firstly define 3rd party services (like SqlServer) and than our services that depend on them